Extensions/anti_capitalism.js

```diff
@@ -28,16 +33,20 @@ XKit.extensions.anti_capitalism = new Object({
 
 	run: function() {
 		this.running = true;
-		
-		if (XKit.extensions.anti_capitalism.preferences.vendor_buttons.value === true) {
+
+		if (XKit.extensions.anti_capitalism.preferences.vendor_buttons.value) {
 	       XKit.tools.add_css(" .post .vendor_button {display: none;}", "anti_capitalism_vendor_buttons");
 	    }
-	    
-	    if (XKit.extensions.anti_capitalism.preferences.sponsored_ads.value === true) {
+
+		if (XKit.extensions.anti_capitalism.preferences.sponsored_posts.value) {
+			XKit.tools.add_css(" .post.sponsored_post {display:none}", "anti_capitalism_sponsored_posts");
```

Comment:
I can't get tumblr to show me sponsored posts anymore, but I know blocking the .sponsored class works as well.